tensorflow/tools/dockerfiles/dockerfiles/arm64v8/devel-cpu-arm64v8.Dockerfile

```diff
@@ -23,8 +23,11 @@ ARG UBUNTU_VERSION=18.04
 
 FROM ubuntu:${UBUNTU_VERSION} AS base
 
+ARG DEBIAN_FRONTEND="noninteractive"
+
 RUN apt-get update && apt-get install -y --no-install-recommends \
         build-essential \
+        clang-format \
```

Comment:
With "clang-format-10` that is available in this Ubuntu 20.04 upgrade we could use `clang-format-10 --dry-run --Werror` if we don't like the automatic pre-commit formatting.